test/pull/expected-stdout

```diff
@@ -12,10 +12,10 @@ INFO	  +  R bar/extractor/keboola.ex-db-mysql/with-rows/rows/disabled
 INFO	  +  R bar/extractor/keboola.ex-db-mysql/with-rows/rows/test-view
 INFO	  +  R bar/extractor/keboola.ex-db-mysql/with-rows/rows/users
 INFO	  +  C bar/other/keboola.variables/shared-code-variables
```

Comment:
V teste su este jedny `variables` (tu ziadna zmena), ... jedna sa o `shared code`, ktory ma `variablesId`, ale nie v configu, ale v row:
https://github.com/keboola/keboola-as-code/blob/aa281381d39bd5b8ad808849936a0e9181e065b0/internal/pkg/fixtures/configs/shared-codes.json#L23-L29

To sa musi vyriesit ako samostatny pripad.